test/e2e/fixtures.go

```diff
@@ -76,7 +76,7 @@ func skipIfIPv6Cluster(tb testing.TB) {
 
 func skipIfNotIPv6Cluster(tb testing.TB) {
 	if clusterInfo.podV6NetworkCIDR == "" {
-		tb.Skipf("Skipping test as it is not needed in IPv4 cluster")
+		tb.Skipf("Skipping test as it is not needed in IPv6 cluster or the IPv6 network CIDR is not set")
```

Comment:
why not copy what we have for `skipIfNotIPv4Cluster`

> Skipping test as it requires IPv6 addresses but the IPv6 network CIDR is not set

I don't think that "Skipping test as it is not needed in IPv6 cluster" is really correct here.

Reply:
Done